AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/SubscriptionSync/Support/EquatableExtensions.swift

```diff
@@ -69,7 +70,7 @@ extension ReconcileAndLocalSaveOperation.State: Equatable {
                 && model1.model.modelName == model2.model.modelName
         case (.executing(let disposition1), .executing(let disposition2)):
             return disposition1 == disposition2
-        case (.notifying(let model1), .notifying(let model2)):
+        case (.notifying(let model1, _), .notifying(let model2, _)):
```

Comment:
add logic to check the lhs and rhs's second parameter to be equal as well

Reply:
added